File: TwoSum/twosum.go
package main

import "fmt"

func main() {
  myArray := [4]int{1,2,3,4}
  tar := 5
  fmt.Println(twoSum(myArray, tar))
}

func twoSum(nums [4]int, target int) [2]int {
  var returnInts [2]int
  for i := 0; i < len(nums); i++ {
    for j := 0; j < len(nums); j++ {
      if nums[i] == nums[j] {
        continue
      } else {
        if (nums[i] + nums[j]) == target {
          returnInts[0] = i
          returnInts[1] = j
          return returnInts
        }
      }
    }
  }
  return returnInts
}

File: TwoSum/twosum.java
public class twosum{
  public static void main(String[] args) {
    int[] numbers = {1,2,3,4};
    int[] output;
    int tar = 5;
    output = twoSum(numbers, tar);
    System.out.println(output);
  } // End of main

  public static int[] twoSum(int[] nums, int target) {
    int[] resultInts = {0,0};
    for(int i = 0; i < nums.length; i++) {
      for(int j = 0; i < nums.length; j++) {
        if(nums[i] == nums[j]) {
          continue;
        } else {
          if((nums[i] + nums[j]) == target) {
            resultInts[0] = nums[i];
            resultInts[1] = nums[j];
            return resultInts;
          }
        }
      }
    }
    return resultInts;
  } // End of TwoSum

}
